### ISSUE TITLE
Signal 7.29 crashes directly when started on iOS 18

### ISSUE THREAD

**Issue by danpfe** [signalapp/Signal-iOS#5875]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
App immideatelly closes on an iOS 18-phone (fresh install on new phone). Seems very similar to https://github.com/signalapp/Signal-iOS/issues/5796, but since that seems fixed it may be a different issue when installing fresh on iOS 18.

### Steps to reproduce
- Install app fresh from App Store
- Tap the app icon

**Actual result:** Crashes/Closes instantly

**Expected result:** App should open

### Device info

**Device**: iPhone 15 Pro Max

**iOS version**: 18.0

**Signal version:** 7.29

**Comment by katosdev** [comment]
I also get the same issue, also on an iPhone 15 Pro Max

**Comment by antoinedesp** [comment]
Same issue, lost all messages conversations and contacts trying to uninstall and reinstall. WTF devs????!!!!

**Comment by danpfe** [comment]
Managed to get out debug log and see that it segfaults seemingly due to accessing an instance of a class that is no (longer) available.

IPS-file in this GIST: https://gist.github.com/danpfe/1abd891c3d8dae3069cc128c0f2ea32a

**Comment by pete-signal** [comment]
The crash report indicates you're running an iOS18 beta - could you update to the released version of iOS 18.0 and see if this helps?

**Comment by danpfe** [comment]
@pete-signal Funny, that did the trick. I did see it also in the IPS-file and I was surprised of it myself. I checked several times in the update-dialog if Beta was activated or if there was an update after seeing it but only yesterday it showed another update.

To the others (@antoinedesp @katosdev) having the same problem: Check again, maybe there was something borked up in the roll-out to your phones as well (or accidentally having Beta activated) and something went wrong when you updated previously. It works for me now.
